diff --git a/dist/kinde-auth-pkce-js.esm.js b/dist/kinde-auth-pkce-js.esm.js
index 6cf84bc5d65411863784575f51acde7af23c5aa7..603a0c8c1bb77241a1d3f2c539d3ae910123bef0 100644
--- a/dist/kinde-auth-pkce-js.esm.js
+++ b/dist/kinde-auth-pkce-js.esm.js
@@ -1753,7 +1753,7 @@ const createKindeClient = async (options) => {
     const isUseCookie = !isDevelopment &&
         !is_dangerously_use_local_storage &&
         C(domain);
-    const isUseLocalStorage = isDevelopment || is_dangerously_use_local_storage;
+    const isUseLocalStorage = Boolean(is_dangerously_use_local_storage);
     // Use LocalStorage from @kinde/js-utils for persistent storage
     const localStorageAdapter = new ce();
     Je(localStorageAdapter);
